lib/Runtime/Library/JavascriptProxy.cpp

```diff
@@ -6,14 +6,19 @@
 
 namespace Js
 {
-    bool JavascriptProxy::IsRevoked() const
+    bool JavascriptProxy::IsRevoked() const noexcept
     {
         return (target == nullptr);
     }
 
+    inline bool JavascriptProxy::IsRevoked(RecyclableObject* object) noexcept
```

Comment:
We probably don't want to add this as a member function, since it's not checking any internal state. I think the existing `IsRevoked` is probably good enough.

Reply:
Done.